scalafix-tests/output/src/main/scala-2/test/noAutoTupling/NoUnitInsertion.scala

```diff
@@ -0,0 +1,24 @@
+package test.noAutoTupling
+
+class NoUnitInsertion {
```

Comment:
so if I understand well:
* this was removed by mistake in https://github.com/scalacenter/scalafix/commit/d5d23dbf37208533254067ddf1cea3dca6904b2b#diff-fd705cc22acdf8f3dc8262c143dd9ff3a6afcdb914b8dc0cd02479258fcdbe3e
* testkit seeing an input without an output is not triggering a failure as it's a valid use-case for linters (`assert` syntax in input file comments)
* The TODOs introduced in https://github.com/scalacenter/scalafix/pull/1118 in `NoAutoTupling` was a subset of this

Reply:
- I didn't know why this test wasn't running. I deleted it by mistake! 
- While grouping the tests together for each rule, I remembered that a small thing was missing to make this rule working. Maybe I will go throw my previous Todo and see what I can fix.